flox-bash/lib/templateFloxEnv/pkgs/default/flox.nix

```
@@ -4,16 +4,31 @@
   # packages.nixpkgs-flox.figlet = {};
   # packages.nixpkgs-flox.bat = { version = "0.22.1"; };
 
+  # Activation Extensions
+  # Provides an extensible `<env>/etc/profile` script.
+  # packages.flox.etc-profiles = {
+  #   # All language packs are installed by default, but you can also
+  #   # select individual packs by uncommenting the line below.
+  #   # Invoke `flox search -c flox etc-profiles -l` to see
```

```suggestion
  #   # Invoke `flox search -l -c flox etc-profiles` to see
```

feel free to ignore, I just think options before arguments looks more "correct"